### DISCUSSION TITLE
Adding volume and auto-fitting the y-axes to both price and volume

### DISCUSSION THREAD

**Discussion by hollowheights** [highfestiva/finplot#410] [General]
I have the code below and it makes for a chart where only the volume is visible - because the values are obviously in the millions.

`    # Call finplot for price data
    fplt.create_plot('My Window', maximize=True)
    fplt.candlestick_ochl(df[['open', 'close', 'high', 'low']])
    fplt.display_timezone = tz.gettz('America/New_York')
    #Call finplot for volume data
    fplt.volume_ocv(df[['open', 'close', 'volume']])`

I have read through the documentation for hours. But I find that:
1) I see examples but no actual documentation of the various functions (?)
2) In the example here: https://github.com/highfestiva/finplot/wiki/Examples there is reference to both a second 'axes' and a second 'plot', but it seems this is a mistake in the documentation. I'm looking to keep it simple and just add volume on top of the one plot I have. It seems to default to 'overlay' but without controlling the axes that doesn't work.

**Comment by highfestiva** [comment]
Hi, sorry I missed your question, that's why it took so long. :)

You have two options: a) put the volume on top of the other plot ("overlay"), but that will not show you the actual numbers on the volume chart when you hover over it; and b) put the volume in a separate axis. Both have examples that you can look at for reference. GL!
